### ISSUE TITLE
Add new configuration but cancel at the end

### ISSUE THREAD

**Issue by dsgwtgfh35y** [Eugeny/tabby#5645]
When the Add New Configuration button is clicked, even if the Cancel button is finally clicked, an empty configuration will still be generated. I think this is unreasonable, please optimize

**Comment by Eugeny** [comment]
Do you mean "configuration" as in "profile"?

**Comment by dsgwtgfh35y** [comment]
> 您是指“配置文件”中的“配置”吗？

yes,The situation also occurs in copy profile,I think when adding or copying for the first time, it should be generated after clicking the confirm button
